qiskit/visualization/circuit_visualization.py

```diff
@@ -441,8 +441,9 @@ def _text_circuit_drawer(circuit, filename=None, reverse_bits=False,
         layout = circuit._layout
     else:
         layout = None
+    phase = circuit.global_phase if hasattr(circuit, 'global_phase') else None
```

Comment:
This might be something good to move to a shared place for the other drawers. But we can handle that when we add this to the other drawers.